src/Home.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+
+
+export default function Home() {
+    return (
+        <View style={styles.container}>
+            <Text >Todo App</Text>
```

Comment:
Nit: empty white space in the opening `Text` tag. Maybe we should add `prettier` to the project to clean these up, thoughts?

Reply:
I already installed it earlier. It was showing up some errors when I tried to format the code. Restarting the VSCode fixed it.